kernel/classes/ezcontentobjecttreenodeoperations.php

```diff
@@ -113,6 +113,7 @@ static function move( $nodeID, $newParentNodeID )
                 $nodeIDList = array( $nodeID );
                 eZSearch::removeNodeAssignment( $node->attribute( 'main_node_id' ), $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList );
                 eZSearch::addNodeAssignment( $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList, true );
+                eZSearch::refreshChildren( $node );
```

Comment:
same comment here, from how I understood it, and based on the call one line above here, we basically need to make sure customer uses this indexing cronjob, OR maybe rather we need to make sure [ezplatformsearch](https://github.com/netgen/ezplatformsearch) handles moved node actions.

Either way we need to update doc somewhere, probably in ezplatformsearch extension.